benchmarks/benchmarks/generate_data.py

```diff
@@ -35,21 +34,25 @@
     raise ValueError(error)
 
 
-def run_function_elsewhere(python_exe, func_to_run, func_call_string):
+def run_function_elsewhere(func_to_run, *args, **kwargs):
     """
-    Run a given function using an alternative python executable.
+    Run a given function using the :const:`DATA_GEN_PYTHON` executable.
```

Comment:
I know I removed the 'python_exe' arg in my version, but it might still make more sense to retain it here.
Your choice, though, really..

Reply:
I flip-flopped a lot on this internally but I finally decided it was an example of me being over-flexible. I don't expect ANY examples of using more than one python executable with this function.